ncl/ncl_entries/general_applied_math.ipynb

```diff
@@ -174,6 +183,67 @@
     "print(avg_output)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## ceil\n",
+    "\n",
+    "NCL's `ceil` function rounds the smallest integer up"
```

Comment:
```suggestion
    "NCL's `ceil` function returns the smallest integral value greater than or equal to each input value"
```

I suggest we use the language from NCL here.  Or if the use of "integral" is too jargony maybe swapping that with "integer".